tests/FakeItEasy.Analyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -114,9 +81,9 @@ private static void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResult
                     {
                         var message =
                             FormattableString.Invariant(
-                                $@"Expected:
-A project diagnostic with No location
-Actual:
+                                $@"Expected:
+A project diagnostic with No location
+Actual:

```

Comment:
Actually, in the template it was using `string.Format` and `\r\n` instead of a verbatim interpolated string... so it must be my fault ;)